packages/stats/gbstats/bayesian/tests.py

```diff
@@ -126,14 +128,18 @@ def chance_to_win(self, mean_diff: float, std_diff: float) -> float:
         else:
             return norm.sf(0, mean_diff, std_diff)  # type: ignore
 
-    def scale_result(
-        self, result: BayesianTestResult, p: float, d: float
-    ) -> BayesianTestResult:
+    def scale_result(self, result: BayesianTestResult) -> BayesianTestResult:
         if result.uplift.dist != "normal":
             raise ValueError("Cannot scale relative results.")
-        if p == 0:
+        if (
+            self.phase_length_days == 0
+            or self.traffic_percentage == 0
+            or not isinstance(self.stat_a, ScaledImpactStatistic)
+        ):
             return self._default_output(ZERO_SCALED_VARIATION_MESSAGE)
```

Comment:
This is the wrong error message for the `not is instance` check above. I know these are opaque/not used right now, but we should create the right message.

Reply:
Also, we should throw an error if the total_users is None.

Reply:
done